mappings/interactionLoggingClientData.mapping

```diff
@@ -0,0 +1 @@
+CLASS aimp interactionLoggingClientData
```

Comment:
Class names should always start with a capital letter ;)

Reply:
Oops,oversight